docs/Plugins-evals.md

```diff
@@ -0,0 +1,241 @@
+
+##### 2.C.2 – Process Discovery (T1057) via Command-Line Interface (T1059)
+The `tasklist` utility is executed via `cmd` to enumerate local running processes.
+
+##### 2.D.1 – System Service Discovery (T1007) via Command-Line Interface (T1059)
+The `sc` utility is executed via `cmd` to enumerate local active services.
+
+##### 2.D.2 – System Service Discovery (T1007) via Command-Line Interface (T1059)
+The `net` utility is executed via `cmd` to enumerate local active services.
+
+##### 2.E.1 – System Information Discovery (T1082) via Command-Line Interface (T1059)
+The `systeminfo` utility is executed via `cmd` to enumerate the local operating system configuration.
+
+##### 2.E.2 – System Information Discovery (T1082) via Command-Line Interface (T1059)
+The `net` utility is executed via `cmd` to enumerate the local operating system configuration.
+
+##### 2.F.1 – Permissions Groups Discovery (T1069) via Command-Line Interface (T1059)
+The `net` utility is executed via `cmd` to enumerate members of the local system's administrators group.
+
+##### 2.F.2 – Permissions Groups Discovery (T1069) via Command-Line Interface (T1059)
+The `net` utility is executed via `cmd` to enumerate members of the domain controller’s administrators group.
+
+##### 2.F.3 – Permissions Groups Discovery (T1069) via Command-Line Interface (T1059)
+The `net` utility is executed via `cmd` to enumerate members of the domain administrators group.
+
+##### 2.G.1 – Account Discovery (T1087) via Command-Line Interface (T1059)
+The `net` utility is executed via `cmd` to enumerate domain user accounts.
+
+##### 2.G.2 – Account Discovery (T1087) via Command-Line Interface (T1059)
+The `net` utility is executed via `cmd` to enumerate detailed information about a specific user account.
+
+##### 2.H.1 – Query Registry (T1012) via Command-Line Interface (T1059)
+The `reg` utility is executed via `cmd` to enumerate a specific Registry key associated with local system policies to ensure that the user will not be prompted for credentials when elevating permissions.
+
+#### Step 3 - Privilege Escalation
+
+Step 3 tested the following techniques:
+
+##### 3.A.1 - Bypass User Account Control (T1088) via Access Token Manipulation (T1134)
+This Eval plugin ability uses [Empire's Invoke-BypassUACTokenManipulation.ps1](https://github.com/EmpireProject/Empire/blob/master/data/module_source/privesc/Invoke-BypassUACTokenManipulation.ps1) script to steal the token of an existing high-integrity process and launch a new, high-integrity CALDERA RAT with limited functionality.
+
+\*Note if a high integrity process is not running the script will attempt to spawn TaskMgr.exe with the RunAs flag. If there is no high integrity process running or TaskMgr.exe fails to spawn as admin the script will fail. We recommend spawning cmd.exe as the high integrity process prior to starting the DIY Eval. Read more about the lab setup [here]()
```

Comment:
Not sure where this link was meant to point to.

Reply:
https://attackevals.mitre.org/methodology/round1/environment.html